public/locales_euid/en.json

```diff
@@ -20,7 +20,7 @@
   "Powered by": "Powered by",
   "Internal Server Error": "Internal Server Error",
   "euid-exit": "EXIT",
-  "Welcome!": "Welcome!",
+  "Welcome!": "Hello,",
```

Comment:
Currently I have only changed the copy of euid for the welcome part. A lot of the copy seems to be EUID specific, so thought best not to mess with the rest, given it wasn't mentioned in the ticket or Figma either.